rook/helmfile.d/values/cluster.yaml.gotmpl

```diff
@@ -0,0 +1,106 @@
+{{- $cluster := .Values.cluster -}}
+
+configOverride: |
+  [global]
+  osd pool default size = 2
```

Comment:
What was the issue with this to begin with, because this should really only affect the metadata pool, right? Since we explicitly set it on pools we create.

Reply:
Yea, I don't remember exactly why. This just set the global default settings for the cluster. Since this is a leftover from when we first setup ceph for pulsen I would assume that it was either because:

1. There was some issue with rook on that version so it didn't properly pick the correct pool size based on the CephCluster object
2. Or that it was to safeguard against a new pool being created without setting the pool size so it would pick this by default (but I don't ever see that happen)

Anyway as I say, we should probably do some test to remove this just to verify that it isn't needed anymore and remove it completely otherwise.

Reply:
I see, well I'll make it configurable and let it default to 2 as we've had earlier.